browser/download/brave_download_commands.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_BROWSER_DOWNLOAD_BRAVE_DOWNLOAD_COMMANDS_H_
+#define BRAVE_BROWSER_DOWNLOAD_BRAVE_DOWNLOAD_COMMANDS_H_
+
+#include "chrome/browser/download/download_commands.h"
+
+// This class overrides the DownloadCommands class to provide Brave-specific
+// command handling. This class is used in DownloadBubbleRowView and
+// DownloadShelfView.
+class BraveDownloadCommands : public DownloadCommands {
+ public:
+  // Create brave specific commands set instead of appending to
+  // DownloadCommands::Command to avoid many upstream changes.
+  // Removes the download item from the list. The actual file is not deleted.
+  // Used by download shelf view.
+  static constexpr Command kRemoveFromList = Command(Command::kMaxValue + 1);
```

Comment:
Not sure if relevant, but the download shelf is being deleted in M140.

https://chromium-review.googlesource.com/c/chromium/src/+/6735129